ckanext/adfs/templates/user/new.html

```diff
@@ -0,0 +1,47 @@
+{% extends "page.html" %}
+
+{% block subtitle %}{{ _('Register') }}{% endblock %}
+
+{% block breadcrumb_content %}
+  <li class="active">{{ h.nav_link(_('Registration'), controller='user', action='register') }}</li>
+{% endblock %}
+
+{% block styles %}
+    {{ super() }}
+    {% resource 'adfs/css/adfs_user.css' %}
+{% endblock %}
+
+{% block primary_content %}
+  <article class="module">
+    <div class="module-content">
+      {% block primary_content_inner %}
+      <section class="module">
+        <div class="module-content">
+           <h1 class="page-heading">CNRA Staff</h1>
+            <p>If you are an employee of a CNRA organization, use the CNRA Log In button on the Log In page to automatically create an account using your enterprise credentials. If it does not work for you, contact the Open Data team for altenate account instructions.</p>
```

Comment:
According to the mockups the text "contact the Open Data team" should be a link.